requirements.txt

```diff
@@ -3,3 +3,4 @@ pyyaml>=3.11,<4
 progressbar>=2.3,<3
 requests>=2.9.1,<3
 plumbum>=1.6.2,<2
+bltest>=1.0.0,<2
```

Comment:
Why do we need it if we're not in development? Dependencies should only ever be in one of the two requirements files, as dev imports prod & the odds of getting version numbers out of sync if we put them in both is too high.